src/main/java/com/VetApp/PurrgentCare/models/Account.java

```diff
@@ -38,11 +37,7 @@ public class Account {
 
     public void addAccountHolder(Person accountHolder) {
         accountHolder.setAccount(this);
-        //List would not allow addition of accountHolders because it is immutable
-        //Had to convert to ArrayList to make it mutable and then set object List
-        List<Person> accountHolders = new ArrayList<>(getAccountHolders());
-        accountHolders.add(accountHolder);
-        setAccountHolders(accountHolders);
+        getAccountHolders().add(accountHolder);
     }
```

Comment:
Making the test work broke the implementation.
Exploring why the test is generating an error while the implementation is not.

Reply:
This looks wrong b/c you're not storing it in an object. getAccountHolders().add(accountHolder) does not actually alter anything. It just adds to it in memory but not in the DB.

Reply:
That is working in Postman 🤷‍♂️
I tried this.accountHolders.add(accountHolder) and it also works in Postman and fails the Test.

From what I have found online it seems like when we make the Account for the test when we setup the List<Person> accountHolders it's making that immutable and so when I run the test I get an error, but in the implementation it works fine.

Reply:
See thread below for resolution